templates/common/copilot-plugin-existing-api-api-key/teamsapp.yml.tpl

```diff
@@ -20,9 +20,14 @@ provision:
   # Register API KEY
   - uses: apiKey/create
     with:
+      # Name of the API Key
       name: {{ApiSpecAuthName}}
```

Comment:
Also need to update the version of the teamsapp.yaml file.

```
# yaml-language-server: $schema=https://aka.ms/teams-toolkit/v1.3/yaml.schema.json
# Visit https://aka.ms/teamsfx-v5.0-guide for details on this file
# Visit https://aka.ms/teamsfx-actions for details on actions
version: v1.3
```

Reply:
Thanks for catching this. Updated